config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :mechanics, only: [:show] 
+  resources :mechanic_rides, only: [:create, :new], controller: :mechanic_rides
```

Comment:
Since the form is rendered on the mechanics show page, you don't need the `:new` path. Also, you shouldn't need the `controller: :mechanic_rides` since that's the name of the resource, I'm curious if that was giving you problems.